### ISSUE TITLE
ETH → DFC: Improve error message in tx error when amt is too low

### ISSUE THREAD

**Issue by izzycsy** [BirthdayResearch/quantum#689]
```
Ver: 1.1.0, 1.1.1
Network: MainNet
Flow: ETH → DFC 
Description: If amt is too low, tx error
```

**Actual:**
Tried sending 0.000002 wBTC from ETH → DFC  Review transaction > Transaction Error <string>

**Expected:**
Tried sending 0.000002 wBTC from ETH → DFC  Review transaction > Transaction Error <more meaningful error message>

**To reproduce:**
1. Select Source Network: Ethereum
2. Input `0.000002` wBTC > select `Review transaction` > Transaction Error <string>

https://user-images.githubusercontent.com/65014479/223404597-d67f99a0-7041-4927-b2f2-c8f33fc181e7.mov

![230307 Quantum ETH   DFC - WBTC   dBTC amt too low2](https://user-images.githubusercontent.com/65014479/223404609-e37964f7-9551-4b1f-b6e8-eef7ebd8173d.png)

**Comment by lykalabrada** [comment]
Can we check if this issue is fixed? I'm guessing this is probably related to this [issue](https://github.com/WavesHQ/quantum/issues/691)

**Comment by izzycsy** [comment]
Issue resolved v1.4.0
